epub33/explainers/EPUB-33-security-privacy.md

```diff
@@ -0,0 +1,103 @@
+4. Recording progress through a book and whether a book was finished. 
+5. Some reading systems track every user reading session, including the time of day, the duration, how many pages were read, what book, the user’s IP address if a web-based reader, etc. 
+
+
+
+### 2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+The EPUB specification does not mention reading system features like bookmarks or highlights. There is no requirement to remember reading state. 
+
+### 3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+The EPUB specifications do not mention personal information.
+
+### 4. How does this specification deal with sensitive information?
+
+Merely the fact that you are reading a particular book can be sensitive information. Again, the specification offers no guidance to reading systems on this matter. 
+
+### 5. Does this specification introduce new state for an origin that persists across browsing sessions?
+
+Reading systems, in order to meet user expectations, must retain some information across browsing sessions, for example to remember reading location. The spec has a non-normative section saying that reading systems should treat each EPUB as a unique origin, but does not say anything about most reading system behaviors.
+
+### 6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
+
+Many EPUB reading systems do not support scripting at all. For those that do, scripts in EPUBs that seek this sort of information might be blocked by the application before they even reach the platform level. If not, the risks are likely similar to that of HTML/CSS. 
+
+
+### 7. Does this specification allow an origin access to sensors on a user’s device?
+
+EPUB does have requirements for user agents to respond to device orientation (`portrait | landscape`) for fixed-layout EPUBs. 
+
+There has been talk about scripted EPUBs using geolocation information; I don’t know if this has happened in the wild yet, or if any reading systems would allow access to this information. This would happen through existing Web APIs; EPUB does not specify new APIs to interact with sensors. 
+
+### 8. What data does this specification expose to an origin? Please also document what data is identical to data exposed by other features, in the same or different contexts.
+
+Again, this depends on the architecture of the reading system. 
+
+### 9. Does this specification enable new script execution/loading mechanisms?
+
+No. Scripting is not forbidden, but follows the HTML model.
+
+### 10. Does this specification allow an origin to access other devices?
+
+EPUB reading systems often implement syncing behaviour across multiple devices with the same user account. This is not required or mentioned by the spec. 
+
```

Comment:
Actually... to be very precise, syncing behaviour across multiple devices for the same EPUB publication. I do not think that there is a way for a script runing in one EPUB publication to access information on another EPUB publication, even if it belongs to the same user account.

Reply:
I think Jiminy has shown that scripts running in one EPUB can indeed access information in other EPUBs.

Reply:
Really? I would consider that a bug (but obviously a privacy/security breach). If the EPUB instance is a single origin, then this could be considered as a CORS violation of some sort...

Reply:
Given that what we say about origins (especially in 3.2 and earlier), this might not even violate the spec.

Reply:
Why? This means that another EPUB instance has definitely a different origin, and that means scripts are not supposed to get there...